assets/src/edit-story/elements/index.js

```diff
@@ -46,6 +48,18 @@ export const createNewElement = (type, attributes = {}) => {
 
 export const createPage = (attributes) => createNewElement('page', attributes);
 
+export const mergeElements = (
+  from,
+  into,
+  keep = Object.keys(StoryElementPropsTypes)
```

Comment:
A runtime part relying on PropTypes?

Reply:
What do you think is the best way to do this without redundancy?

Reply:
I think we may have to live with some redundancy here. May be some sort of globally shared `universalProperties` or such in consts. And we would need to add a unit test that checks that the two sets are consistent: this const and the `StoryElementPropsTypes`.